server/index.js

```diff
@@ -54,7 +57,12 @@ io.on('connection', (socket) => {
     // received signal to start a session from the host of a room, emit redirect signal to all guests and host
     socket.on('begin', ( user , callback) => {
         console.log('begin session signal received')
-        io.to(user.room).emit('sessionMembers', { room: user.room, users: getUsersInRoom(user.room), host: getHost(user.room)});
+
+        // get list of top 10 movies to send to all guests of the room
+        console.log('getting top 10')
```

Comment:
Should the redis initializeRoom event be triggered prior to moving users to swiping page?

Reply:
The initialize room event can be called in this .then block by passing in the returned top 10 list of movies , however it isn't in main yet. I'll add a comment indicating a good place to call the redis initializeRoom function in this code block!